README.md

```diff
@@ -35,7 +38,10 @@ var add = function (x, y) {
 }
 ```
 
--- YOUR ANSWER HERE --
+ES5.
+NO.
```

Comment:
This is anonymous. The function is assigned to a variable called add, but has no name itself